pkg/gitlab/groupreadwriter.go

```diff
@@ -0,0 +1,358 @@
+// Copyright 2024 The Authors (see AUTHORS file)
```

Comment:
Why is there so much conversion between the string and integer IDs? It looks like the library accepts an `any` and [handles the conversion](https://github.com/xanzy/go-gitlab/blob/main/gitlab.go#L946) anyway. Let's not do all this extra work that conflates the business logic.

Reply:
Oops, fixed that. The API also accepts full paths (e.g. `foo/bar`) as a group ID and I mistakenly thought that it assumes all strings you pass are paths and not IDs.